content/docs/intro/concepts/runtime-functions.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Runtime Functions"
+meta_desc: A quick look at runtime helper functions for Pulumi.
+menu:
+  intro:
+    parent: concepts
+    weight: 10
+---
+
+The Pulumi SDK library includes a number of helper functions for the following actions:
+
+- Getting information about the current deployment
+- Using logging information for diagnostics, warnings, or errors pertaining to the deployment
+- Serializing Lambdas to turn JavaScript callbacks into data, which can be used as application code
+
+## Getting the Current Project or Stack
+
+The [`get_project`]({{< relref "/docs/reference/pkg/python/pulumi#pulumi.get_project" >}}) and [`get_stack`]({{< relref "/docs/reference/pkg/python/pulumi#pulumi.get_stack" >}}) functions give you the currently deployed project and stack, respectively. This can be useful for naming or tagging resources.
+
+```python
+project = pulumi.get_project()
+stack = pulumi.get_stack()
+```
+
+## Logging
+
+The [`pulumi.log.debug/info/warn/error`]({{> relref "/docs/reference/pkg/python/pulumi#logging" }}) functions allow you to log diagnostics, warnings, or errors with the Pulumi engine. These are displayed, alongside all other Pulumi output, in the CLI and in the Pulumi Console. They are also logged and stored in case you want to audit or diagnose a past event. Here are the error functions:
+
+```text
+log.info("message")
+log.info("message", resource)
+log.debug("hidden by default")
+log.warn("warning")
+log.error("fatal error")
+```
+
+## Serializing Lambdas (Node.js only)
+
+You can create libraries and components that allow the caller to pass in JavaScript callbacks that are invoked at runtime. For example, you can create an AWS Lambda function or an Azure function by providing a JavaScript callback that serves as its implementation.
+
+> Note: Runtime code provided via callbacks are currently not supported in Python. See [github issue 1535](https://github.com/pulumi/pulumi/issues/1535).
```

Comment:
Particularly odd that this ended up here - it's meant to be in a language chooser so that this is only displayed to users who select Python.  It's also not supported in other language.

Reply:
This is fixed as part of the latest tech review changes.